zuul.d/projects.yaml

```diff
@@ -96,6 +96,19 @@
       - integrated-queue
       - publish-to-galaxy
 
+- project:
+    name: github.com/ansible-collections/community.asa
+    third-party-check:
```

Comment:
lets gate this project with zuul, or is there an issue doing that?

Reply:
+1

Reply:
okay, this is not correct then. We need to remove this, and use the folowing
```
- project:
    name: github.com/ansible-collections/community.asa
    merge-mode: squash-merge
    templates:	    templates:
      - system-required
      - publish-to-galaxy

Reply:
cool, I have made the changes